ClassLibrary/ClassLibrary/Library.cs

```diff
@@ -21,5 +21,46 @@ public class Library
 
         public Reader GetReader(int IDReader) => departmentReaders.GetReader(IDReader);
 
+
+        public void ReaderBoroweBook(int IDReader, String NameBook)
+        {
+            if (departmentReaders.GetReader(IDReader)==null)
+            {
+                throw new ArgumentException($"Not Exist Reader with (ID={IDReader}) in DepartmentReaders");
+            }
+
+            if (NameBook.IsNullOrWhiteSpace())
+            {
+                throw new ArgumentNullException("","Not set Namebook");
```

Comment:
Please check the parameters again and think how you can use the exception in the best way.

Reply:
correct